modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/application/management/v1/ApplicationManagementUserRegistrantSuccessTest.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.integration.test.rest.api.server.application.management.v1;
+
+import io.restassured.response.Response;
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpStatus;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.wso2.identity.integration.test.rest.api.server.application.management.v1.Utils.assertNotBlank;
+import static org.wso2.identity.integration.test.rest.api.server.application.management.v1.Utils.extractApplicationIdFromLocationHeader;
+
+import java.io.IOException;
+
+/**
+ * Test class for Application Management User Registrant Success Test.
+ */
+public class ApplicationManagementUserRegistrantSuccessTest extends ApplicationManagementBaseTest {
+
+    private String createdAppId;
+    static final String APPLICATION_USER_REGISTRANT_CONTEXT = "/user-registrants";
+
+    @Factory(dataProvider = "restAPIUserConfigProvider")
+    public ApplicationManagementUserRegistrantSuccessTest(TestUserMode userMode) throws Exception {
+
+        super(userMode);
+    }
+
+    @BeforeTest(alwaysRun = true)
+    public void initTestClass() throws IOException {
+
+        super.init();
+    }
+
+    @AfterTest(alwaysRun = true)
+    public void testFinish() {
+
+        super.testFinish();
+    }
+
+    @Test
+    public void testRetrieveUserRegistrantsOfInvalidApp() {
+
+        Response response = getResponseOfGet(APPLICATION_MANAGEMENT_API_BASE_PATH + PATH_SEPARATOR + "some-wrong-id" +
+                APPLICATION_USER_REGISTRANT_CONTEXT);
+
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("totalResults", equalTo(0))
+                .body("userRegistrants.size()", equalTo(0));
+    }
+
+    @Test
+    public void testCreateAppWithNoAuthenticationOptions() throws Exception {
```

Comment:
Shall we do this also like doPutAuthenticatorsAndVerify. Is there a need to add this as a test?

Reply:
Is there any disadvantage in having it as a test?

Reply:
No disadvantage. That particular test is not relevant to this context